backend/utils/email.py

```diff
@@ -0,0 +1,33 @@
+from celery import shared_task
+from django.contrib.auth.models import Group
+from django.core.mail import send_mail as django_send_mail
+from django.template.loader import get_template
+
+
+@shared_task(name="utils.send_mail")
+def send_mail(subject, recipient_list, message=None, html_message=None):
+    if recipient_list is None:
```

Comment:
When will this be the case? I feel like if this is ever `None` we shouldn't fail silently like this (the return value of this function is never checked)

Reply:
I did this because I thought this could just be "intended bevahior" for this function for sake of ease lol.

A case where we need this right now is if a post without a user attached onto it (so old posts) are updated, the recipient list would be None. 

I guess the real question is whether this should be the intended behavior of this function. I have no preference. We can throw an error instead and do the None checking on the side of the caller of this function instead

Reply:
I think I would prefer erroring here. Definitely will save a few minutes if someone needs to debug this code later on